doc/nrf/ug_multi_image.rst

```diff
@@ -3,160 +3,120 @@
-the device must usually consist of not one but multiple
-executables that chain-load, or boot, each other. This section
-describes how to build and configure such a Zephyr application.
-
-When and why to use multiple images
-===================================
-
-First off, you need a basic understanding of what exactly an
-executable is and why we would want to have more than one. An
-executable can be referred to by other names depending on context,
-like program, image, or elf file. We will use the name *image*. You
-probably have an idea of what an image is by reading the names, but if
-not: An image consists of pieces of code and data that is identified
-by image-unique names recorded in a single symbol table. The symbol
-table exists as metadata in a ``.elf`` or ``.exe`` file and is not
-included when the image is converted to a HEX file for
-programming. Object files also consist of symbols, code, and data, but
-object files are not images because unlike object files, images have
-their code and data placed at addresses by a linker. In the end, it is
-the linker that creates images. Therefore, if you want to determine if
-you have zero, one, or more images, count the number of times the linker
-has been run.
-
-From this definition, using two images gives you two opportunities:
-Firstly, it allows you to run the linker multiple times and in this
-process partition the final firmware into several regions, something
-that is often desired by bootloaders. And secondly, since there are
-multiple symbol tables, it allows the same symbol name to exist
-multiple times in the final firmware. This again is useful for
-bootloader images, as they often need to have their own fixed copies
-of the same libraries that the application image is using.
-
-Enabling and configuring multiple images
-========================================
-
-Images are organized in a tree of parent-child relationships, which
-means that a parent image requires one or more child images and
-enables them through Kconfig. Usually, the tree is branchless: a
-linked list of images where each image enables the child image that
-boots it. The simplest and most common organization is a single
-application image that enables a single bootloader image.
-
-When a parent image enables a child image through Kconfig, both images are
-configured when CMake is run.
-When configuring the parent image, you can decide how the child image
-should be handled:
-
-* Build the child image from source while building the parent image.
-* Use a prebuilt HEX file of the child image.
-* Ignore the child image.
+In many cases, the firmware that is programmed to a device consists of not only one application, but several separate images, where one of the images (the *parent image*) requires one or more other images (the *child images*) to be present.
+The parent image then *chain-loads* (or *boots*) the child image, which in turn might boot another child image.
```

Comment:
It's the other way around, children boot their parents.

app = parent
bootloader = child

A bootloader boots an application.

Reply:
Also, currently some children boot their siblings.

Technically, there is nothing preventing a parent from booting a child, it's just not been done yet, and there is no indication it will be done in the future either.

Reply:
These comments looks weird out of context :D

Reply:
Updated to:
The child image then *chain-loads* (or *boots*) the parent image, which in turn might be a child image to another parent image and boot that one.